StringConcatenate.txt

```diff
@@ -0,0 +1,9 @@
+/*
+
+Build myStr over several lines by concatenating these two strings: 'This is the first sentence'. and 'This is the second sentence'. using the += operator.
+
+*/
+
```

Comment:
Why is javascript code in .txt file?

Reply:
JS files are not running in my local IDE, maybe because they are snippets. So, started creating .txt file. I will change them to .js.